resources/assets/less/app.less

```diff
@@ -1,64 +1,67 @@
+@import "./tailwind";
+
+@import "./github.less";
+
 html {
-    box-sizing: border-box;
+    @apply .font-sans;
 }
-*, *:before, *:after {
-    box-sizing: inherit;
+body {
+    @apply .mx-auto .bg-grey-lightest;
+    max-width: 1440px;
 }
-
-@import "vendor/bootstrap/bootstrap";
-@import "utilities";
-@import "github";
-@import "components/gistlog";
-@import "components/comment";
-@import "components/notice";
-
-@btn-font-weight: 300;
-@font-family-sans-serif: "Roboto", Helvetica, Arial, sans-serif;
-@font-family-monospace: Consolas, Menlo, Monaco, "Courier New", monospace;
-
-/**
- * Bootstrap-esque
- */
-
-/**
- * Custom
- */
-
-body, label, .checkbox label {
-    font-weight: 300;
+.avatar {
+    @apply .rounded-full .overflow-hidden .relative .mx-auto;
+    width: 5.5rem;
+    height: 5.5rem;
 }
-
-img {
-    max-width: 100%;
+.btn-github {
+    @apply .flex .bg-grey-darkest .text-sm .px-3 .pt-1 .rounded .my-4;
+    color: #fff !important;
 }
-
-.logo {
-    height: 1.2em;
+.gistlog {
+    @apply .bg-white .shadow;
+    margin-top: -2.6rem;
 }
-
-.container {
-    max-width: 730px;
-    position: relative;
+.gistlog__container {
+    max-width: 935px;
 }
-
-.profile-pic {
-    display: block;
-    margin: 0 auto 1.5em;
-    width: 10em;
-
-    img {
-        border: 5px solid #eee;
-        border-radius: 5em;
-        height: 10em;
-        width: 10em;
+.gistlog__content {
+    @apply .mx-auto .block .text-grey .w-full;
+    max-width: 34rem;
+    a, li, ol, p {
```

Comment:
Our general rule at Tighten is: newlines before every brace.

So this would become:

```less
.gistlog__content {
    @apply .mx-auto .block .text-grey .w-full;
    max-width: 34rem;
    
    a, li, ol, p {
        line-height: 1.65
    }
```